### DISCUSSION TITLE
In-App browser dApps - React Native

### DISCUSSION THREAD

**Discussion by naphatswd** [WalletConnect/walletconnect-monorepo#2434] [Developer Support]
Hello guys! I'm currently developing a React Native app with WalletConnect. I've managed to connect with dApps using URI, but I'm facing some challenges integrating it with an in-app browser. If any of you have a guide or suggestions to assist me, I would greatly appreciate it.

**Comment by chendinh** [comment]
what's your problem?

**Reply by chendinh** [reply]
with Browser Dapp, Its good if dapp client which supported @web3modal/standalone for v2.

**Comment by finessevanes** [comment]
Can you share more of the problems you have? Implementation code, error codes, and the SDK you're using would be great!

**Comment by naphatswd** [comment]
no no I didn't have issue with wallet connect (yet 😅) 
what I meant is that I want to find a guide on how to achieve the following scenario: open dApps in the in-app browser and have Wallet Connect prompt to connect.

**Comment by ablankstory** [comment]
@naphatswd - were you able to figure this out? also interested in this type of flow. Thanks!
